packages/amplify-category-xr/Readme.md

```diff
@@ -0,0 +1,12 @@
+# Amplify CLI XR Plugin
+
+## Commands Summary
+
+The current set of commands supported by the Amplify XR Category Plugin
+
+| Command              | Description |
+| --- | --- |
+| amplify xr add | Takes you through a CLI flow to add an xr resource to your backend   |
```

Comment:
Capitalize XR maybe?

Reply:
Good call. I'll change this in the readme and command description.